dockstore-integration-testing/src/test/java/io/dockstore/webservice/TokenResourceIT.java

```diff
@@ -168,13 +168,166 @@ public void getGoogleTokenNewUser() {
         // Check that the token has the right info but ignore randomly generated content
         Token fakeExistingDockstoreToken = getFakeExistingDockstoreToken();
         // looks like we take on the gmail username when no other is provided
-        Assert.assertEquals("potato@gmail.com", token.getUsername());
+        Assert.assertEquals(GOOGLE_ACCOUNT_USERNAME, token.getUsername());
         Assert.assertEquals(fakeExistingDockstoreToken.getTokenSource().toString(), token.getTokenSource());
         Assert.assertEquals(100, token.getId().longValue());
         checkUserProfiles(token.getUserId(), Arrays.asList(TokenType.GOOGLE_COM.toString()));
         verify(GoogleHelper.class);
     }
 
+    /**
+     * Covers case 1, 3, and 5 of the 6 cases listed below. It checks that the user to be logged into is correct.
```

Comment:
As discussed, this format is a bit unclear. 
I think the content of the table is correct, but it was hard to understand